pages/apim/3.x/user-guide/publisher/import-apis.adoc

```diff
@@ -0,0 +1,150 @@
+= Import APIs
+:page-sidebar: apim_3_x_sidebar
+:page-permalink: apim/3.x/apim_publisherguide_import_apis.html
+:page-folder: apim/user-guide/publisher
+:page-layout: apim3x
+:page-liquid:
+
+== Overview
+
+The following sections describe the process of importing an API from its JSON definition.
+
+== API definition
+
+=== What is an API definition ?
+
+An API definition is a JSON representation of your API, and its content (plans, pages, metadata, etc...).
+
+=== How to get an API definition ?
+
+You can get the definition of your API, by exporting it from the APIM console.
+Or, by using the export endpoint
+`**GET /apis/{api.id}/export**`
+(link:/apim/3.x/management-api/3.16/#operation/exportApiDefinition[API reference^]).
+
+=== What does it contain ?
+
+In this definition, each entity (API, plan, or page) has :
```

Comment:
Maybe we should state more accurately what entity contains a cross ID or not (e.g. group, members, roles do not) ?